objax/functional/core/ops.py

```diff
@@ -55,3 +55,22 @@ def upscale_nn(x: JaxArray, scale: int = 2) -> JaxArray:
     x = x.reshape(s[:2] + (s[2], 1, s[3], 1))
     x = jn.tile(x, (1, 1, 1, scale, 1, scale))
     return x.reshape(s[:2] + (scale * s[2], scale * s[3]))
+
+
+def upsample_2d(x: JaxArray, scale: tuple=(2,2), method: str = 'bilinear') -> JaxArray:
```

Comment:
Order APIs alphabetically too please.

Reply:
For `scale`, you could use IntOrTuple to make it easier for users.
See `k` for example in Conv2D https://github.com/google/objax/blob/master/objax/nn/layers.py#L144

Reply:
For method use an EnumOrString, see `padding` argument in Conv2D for inspiration: https://github.com/google/objax/blob/master/objax/nn/layers.py#L148